ddtrace/contrib/flask/monkey.py

```diff
@@ -0,0 +1,487 @@
+import flask
+import os
+import werkzeug
+from wrapt import wrap_function_wrapper as _w
+
+from ddtrace import config, Pin
+
+from ...ext import AppTypes
+from ...ext import http
+from ...propagation.http import HTTPPropagator
+from ...utils.wrappers import unwrap as _u
+from .helpers import get_current_app, get_current_span, get_inherited_pin, func_name, simple_tracer, with_instance_pin
+from .wrappers import wrap_function, wrap_signal
+
+
+# Configure default configuration
```

Comment:
yeah, I changed this because I was trying to do `from ddtrace.contrib.patch import flask_version` for test cases and it was failing due to the naming conflict.

However, you make a good point, I should probably just move `flask_version` to a different file rather than rename `patch.py` to `monkey.py`

Reply:
Ah, **this** was why I did this:

https://github.com/DataDog/dd-trace-py/blob/25643f553fc11ab30644d1fc97235293ea72cc52/tests/contrib/flask/test_idempotency.py#L59-L60

Because otherwise we get `"_w" is not a property of <function patch>`


I'll think of something. I'd prefer to use patch on `_w` here to assert we did patching rather than generate spans, etc.

Reply:
Found a workaround. Don't love it... but works.

```python
with require_modules(required_modules) as missing_modules:
    if not missing_modules:
        # DEV: We do this so we can `@mock.patch('ddtrace.contrib.flask._patch.<func>')` in tests
        from . import patch as _patch
        from .middleware import TraceMiddleware

        patch = _patch.patch
        unpatch = _patch.unpatch

        __all__ = ['TraceMiddleware', 'patch', 'unpatch']
```